app/models/vm_or_template.rb

```diff
@@ -804,8 +804,9 @@ def parent_blue_folder
   alias_method :owning_blue_folder, :parent_blue_folder
 
   def parent_blue_folders(*args)
-    f = parent_blue_folder
-    f.nil? ? [] : f.folder_path_objs(*args)
+    with_relationship_type('ems_metadata') do
+      parents_scope(:of_type => "EmsFolder").folder_path_objs("ems_metadata", *args)
```

Comment:
`EmsFolder` has:
```ruby
folder_path_resource_ids = folder_rels.path.where(:resource_type => "EmsFolder",
                                                  :relationship  => relationship_type).select(:resource_id)
```

The `folder_rels` is called on `parents_scope(:of_type => "EmsFolder")`